### DISCUSSION TITLE
Welcome to WoWTools Discussions!

### DISCUSSION THREAD

**Discussion by husandro** [husandro/WoWTools#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Darksky1991** [comment]
感觉要火，老哥加油，老哥写个readme啊

**Reply by AKILLAR185** [reply]
can you update wow midnigth with wowtool i love it so much

**Reply by husandro** [reply]
不写，这年头，谁还看说明

**Comment by AKILLAR185** [comment]
HELLO, CAN YOU UPDATE TO midnight WOWTOOL PLEASE

**Reply by husandro** [reply]
## Interface: 120001
https://www.curseforge.com/wow/addons/wowtools
<img width="1055" height="120" alt="QQ20260327-102859" src="https://github.com/user-attachments/assets/cc0ff799-15d5-439b-a999-9590148f7a61" />
